loadgen/powers-shim.js

```diff
@@ -0,0 +1,39 @@
+import path from 'path';
+
+import { E } from '@agoric/eventual-send';
+
+const filename = new URL(import.meta.url).pathname;
+const dirname = path.dirname(filename);
+/** @param {string[]} paths */
+export const pathResolveShim = (...paths) => path.resolve(dirname, ...paths);
+
+/**
+ * Create and save the loadgen kit if necessary
+ *
+ * @param { ERef<Pick<import('./types').Home, 'zoe'>> } home
+ * @param { import('./types').DeployPowers } powers
+ */
+export const makeInstall = async (home, powers) => {
+  try {
+    const { makeHelpers } = await import('@agoric/deploy-script-support');
+    const { install } = await makeHelpers(home, powers);
+    return install;
+  } catch (e) {
```

Comment:
Thinking about this, I'd like to not regress this case for newer revisions. Does anyone have a suggestion? Maybe an `env` variable set by the integration test of `agoric-sdk` ?

Reply:
I don’t have a suggestion. I’m surprised that there are versions old enough that `install` or the deploy script helpers are absent.

Reply:
Not absent, `@agoric/deploy-script-support` goes back 2 years, and `install` was always part of it. But until https://github.com/Agoric/agoric-sdk/pull/6852 lands, recent revisions fail to import using `esm`, which would result in a rejection on dynamic import. I think I basically want something like
```js
if (env.MUST_USE_PUBLISH_BUNDLE) {
  Fail`Couldn't load deploy-script-support for publishing bundle`;
}
```

With `MUST_USE_PUBLISH_BUNDLE` being set by the CI script starting in https://github.com/Agoric/agoric-sdk/pull/6852

Reply:
Added just this